examples/lib/stories/collision_detection/circles.dart

```diff
@@ -0,0 +1,75 @@
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/game.dart';
+import 'package:flame/geometry.dart';
+import 'package:flame/gestures.dart';
+import 'package:flame/palette.dart';
+import 'package:flutter/material.dart' hide Image, Draggable;
+
+class MyCollidable extends PositionComponent
+    with HasGameRef<Circles>, Hitbox, Collidable {
+  late Vector2 velocity;
+  final _collisionColor = Colors.amber;
+  bool _isWallHit = false;
+  bool _isCollision = false;
+
+  MyCollidable(Vector2 position)
+      : super(
+          position: position,
+          size: Vector2.all(100),
+          anchor: Anchor.center,
+        ) {
+    addShape(HitboxCircle());
+  }
+
+  @override
+  Future<void> onLoad() async {
+    final center = gameRef!.size / 2;
+    velocity = (center - position).normalized() * 150;
```

Comment:
is `normalized() * x` the same as `scaleTo(x)` ?

Reply:
Ah, good catch!